controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -256,6 +266,45 @@ def test_delete_snapshot_succeeds(self):
         self.mediator.client.cmd.snapshot_delete = Mock()
         self.mediator.delete_snapshot("snap-wwn")
 
+    def test_get_object_by_id_return_correct_snapshot(self):
+        snap_name = "snap"
+        snap_vol_name = "snap_vol"
+        xcli_snap = self._get_single_snapshot_result_mock(snap_name, snap_vol_name)
+        self.mediator.client.cmd.vol_list.return_value = xcli_snap
+        res = self.mediator.get_object_by_id(snap_name, "snapshot")
+        self.assertTrue(res.name == snap_name)
+        self.assertTrue(res.volume_name == snap_vol_name)
+
+    def test_get_object_by_id_return_correct_volume(self):
+        volume_name = "volume_name"
+        vol = utils.get_mock_xiv_volume(10, volume_name, "wwn")
+        ret = Mock()
+        ret.as_single_element = vol
+        self.mediator.client.cmd.vol_list.return_value = ret
```

Comment:
if you're not going to use meaningful names, then at least combine these lines into one line

Reply:
done